src/applications/disability-benefits/686c-674/config/chapters/stepchild-no-longer-part-of-household/stepchild-information/stepchild-information.js

```diff
@@ -47,18 +47,18 @@ export const uiSchema = {
       whoDoesTheStepchildLiveWith: {
         'ui:title': 'Who does this stepchild live with?',
         first: {
-          'ui:title': 'First Name',
+          'ui:title': 'First Name of parent or guardian',
```

Comment:
I think `Name` should be lowercased to `name` to maintain consistency with the other workflows.

Reply:
true, updated